examples/k8s-bookinfo.yaml

```diff
@@ -216,43 +201,31 @@ spec:
         ports:
         - containerPort: 6379
         env:
-        - name: A8_REGISTER
-          value: "true"
         - name: A8_PROXY
           value: "true"
-        - name: A8_REGISTRY_URL
-          value: http://$(REGISTRY_SERVICE_HOST):$(REGISTRY_SERVICE_PORT)
+        - name: A8_REGISTRY_BACKEND
+          value: kubernetes
         - name: A8_CONTROLLER_URL
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_CONTROLLER_POLL
           value: 5s
-        - name: A8_REGISTRY_POLL
-          value: 5s
         - name: A8_LOGSTASH_SERVER
           value: $(LOGSERVER_SERVICE_HOST):8092
         - name: A8_SERVICE
-          value: reviews:v2
-        - name: A8_ENDPOINT_HOST
-          valueFrom:
-            fieldRef:
-              fieldPath: status.podIP
-        - name: A8_ENDPOINT_PORT
-          value: "9080"
+          value: reviews:version=v2
```

Comment:
Also, have you tried this with Gremlin tests? I am pretty sure it will fail, because if we change the service versions, we need to update the bookinfo-topology.json, bookinfo-gremlins.json and bookinfo-checklist.json files. They refer to these services as `reviews:v2` instead of `reviews:version=v2`.